packages/core/ios/RNEmbrace/EmbraceManager.m

```diff
@@ -84,6 +84,41 @@ @interface RCT_EXTERN_MODULE(EmbraceManager, NSObject)
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 
+RCT_EXTERN_METHOD(startSpan:(NSString *)name
+                  parentSpanId:(NSString *)parentSpanId
+                  startTimeMS:(nonnull NSNumber *)startTimeMS
```

Comment:
minor nit: could this be `startTimeMs`? If that's inconsistent with how naming has been done then it's not that important

Reply:
yep let me update